PMSCore/ProjectClass.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PMSCore {
+    //class Task with attributes: Name, Description, Status; private attribute Status demonstrates encapsulation
+    public class Task { 
+        public string Name { get; set; } //naming the Task
+        public string Description { get; set; } //providing a brief description of the Task
+        public string Status { get; private set; } = "ToDo"; //initial Status is ToDo, it is protected (only accessible from within the class and derived classes)
+
+        public Task(string name, string description) //constructor part, to initialize a new task, hiding internal components of the Task => abstraction   
+        {
+            Name = name; //setting the name for it
+            Description = description; //providing a description
+        }
+
+        public void UpdateStatus(string newStatus) //method that updates a new value for this task => abstraction (status can't be changed directly)
+        {
+            if (newStatus == "ToDo" || newStatus == "In Progress" || newStatus == "Completed") //update status only if it matches one of these: "ToDo", "In Progress", "Completed"
+                Status = newStatus;
+        }
+    }
+    //class Project which is a group of tasks, also its attributes and methods are another example of Abstraction
```

Comment:
Please let's keep the class comment documentation unifrom please see this example:
https://github.com/DistroRickDev/PMS-Core/pull/7/files
Also no need to inline comment every code line the code speaks for itself